projects/libreoffice/build.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash -eu
+
+git pull -r
+
+#shuffle CXXFLAGS -stdlib=libc++ arg into CXX as well because we use
+#the CXX as the linker and need to pass -stdlib=libc++ to build
+export CXX="$CXX -stdlib=libc++"
+#similarly force the -fsanitize etc args in as well as pthread to get
+#things to link successfully during the build
+export LDFLAGS="$CFLAGS -lpthread"
+
+cd $OUT
+#build under $OUT cause its the only place without enough space to do so
+#build in a sub scratch dir so we can easily throw away the build artifacts
+#afterwards
+mkdir scratch
+cd scratch
+$SRC/libreoffice/autogen.sh --with-distro=LibreOfficeOssFuzz
+
+make build-nocheck
```

Comment:
what here is building the fuzz target(s) and links them with the fuzzing engine? 
Note that OSS-Fuzz will (soon) support more than one fuzzing engine, and so this build script should not 
assume just libFuzzer.